pages/docs/environments/gitpod.mdx

```diff
@@ -10,10 +10,69 @@ Gitpod is an open-source application that provides a prebuilt development enviro
 
 You can use Gitpod in the cloud. Please note, that Gitpod requires additional configuration to work on the authenticated pages in LinkFree. See instructions for [how to set up your OAuth app](/docs/environments/environment-variables) for more information.
 
-There are 2 ways you can open a repository with Gitpod:
+There are **2** ways you can open a repository with Gitpod.
+
+####1st Method: Prefix the url with `gitpod.io/#`
+
+- Go to the GitHub page of your forked LinkFree repository
+- Copy the url from the address bar and paste the code `gitpod.io/#` in front of it
+&nbsp;
+  ![Screen1-Add snippet in front of the url](https://github.com/JenIsCoding/img_for_projects/assets/107807044/90d0b890-ac6f-4ce2-9544-4ebcae699ea2)
+&nbsp;
+- After starting the search, a warning screen will open; you can click the *Continue* button
+&nbsp;
+![Screen2-Pop up warning](https://github.com/JenIsCoding/img_for_projects/assets/107807044/e4c08e0d-8425-4827-91d6-961946cdd312)
+&nbsp;
+- The VS Code environment will appear
+&nbsp;
+![Screen3-VS Code environment](https://github.com/JenIsCoding/img_for_projects/assets/107807044/76e85438-f637-46d0-ab4a-b35e1ed813f2)
+&nbsp;
+
+#### 2nd Method: Use a browser extension
+The Gitpod extensions are available for [Firefox](https://addons.mozilla.org/it/firefox/addon/gitpod/) and [Chrome](https://chrome.google.com/webstore/detail/gitpod-always-ready-to-co/dodmmooeoklaejobgleioelladacbeki) (also works for Edge, Brave and other Chromium-based browsers).
+For further details, read the [Browser Extension documentation](https://www.gitpod.io/docs/configure/user-settings/browser-extension) on Gitpod official website.
 
-- Prefix the url with `gitpod.io/#`
-- Use the [Browser Extension](https://www.gitpod.io/docs/configure/user-settings/browser-extension)
+### Permissions
+If you encounter permission issues, follow the next steps:
+
+- Click on the menu button (the three horizonal lines icon) in the upper left corner and open the Dashboard
```

Comment:
Same thoughts here re: unordered vs ordered list.  It may be more clear to have numbered steps here.

Reply:
Same here as well!